apiserver/billing/odoostorage/odoo/odoo8/odoo8.go

```diff
@@ -285,8 +286,17 @@ func odooIDToK8sID(id int) string {
 	return fmt.Sprintf("be-%d", id)
 }
 
-func mapPartnersToBillingEntity(company model.Partner, accounting model.Partner) billingv1.BillingEntity {
+func mapPartnersToBillingEntity(ctx context.Context, company model.Partner, accounting model.Partner) billingv1.BillingEntity {
+	l := klog.FromContext(ctx)
 	name := odooIDToK8sID(accounting.ID)
+
+	var status billingv1.BillingEntityStatus
+	err := json.Unmarshal([]byte(accounting.Status.Value), &status)
+
+	if err != nil {
+		l.Info("Could not unmarshal BillingEntityStatus", "billingEntityName", name)
```

Comment:
We definitely want to log the error and the status JSON here. Arguably, this should also use `l.Warning()` instead of `l.Info()`.

```suggestion
		l.Warning("Failed to unmarshal BillingEntityStatus", "billingEntityName", name, "raw status", accounting.Status.Value, "error", err)
```

Reply:
While I agree with you, I unfortunately found that there is no warning, only `Info` and `Error`. But I do suppose `Error` makes more sense.